src/compiler/commandLineParser.ts

```diff
@@ -1258,125 +1258,12 @@ namespace ts {
         return { buildOptions, projects, errors };
     }
 
-    function getDiagnosticText(_message: DiagnosticMessage, ..._args: any[]): string {
+    /* @internal */
+    export function getDiagnosticText(_message: DiagnosticMessage, ..._args: any[]): string {
         const diagnostic = createCompilerDiagnostic.apply(undefined, arguments);
         return <string>diagnostic.messageText;
     }
 
-    /* @internal */

```

Comment:
did these change when moved?

Reply:
Moved the functions to `executeCommandLine.ts` since that's where they were used anyways